bedrock/mozorg/templates/mozorg/about/governance/policies/commit/access-policy.html

```diff
@@ -87,7 +91,7 @@ <h4>{{ _('Hg') }}</h4>
 
 <h4 id="L10n">{{ _('L10n') }}</h4>
 
-<p>{{ _('In addition, l10n is a separate category so l10n-only access can be more freely given than might be the case if it were included in level 2. This exception is worth making because of the size and diversity of the l10n community and the looser relationship people in it sometimes have to the rest of the project. l10n access implies level 1 access but not level 2 access. The named vouchers are Axel Hecht and Jeff Beatty.') }}</p>
+<p>{{ _('In addition, l10n is a separate category so l10n-only access can be more freely given than might be the case if it were included in level 2. This exception is worth making because of the size and diversity of the l10n community and the looser relationship people in it sometimes have to the rest of the project. l10n access implies level 1 access but not level 2 access. <!--The named vouchers are Axel Hecht and Jeff Beatty.') }}--></p>
```

Comment:
Nit: It works as it is, but cleaner and safer to put the `}}` outside of the HTML comment 

```suggestion
<p>{{ _('In addition, l10n is a separate category so l10n-only access can be more freely given than might be the case if it were included in level 2. This exception is worth making because of the size and diversity of the l10n community and the looser relationship people in it sometimes have to the rest of the project. l10n access implies level 1 access but not level 2 access. <!--The named vouchers are Axel Hecht and Jeff Beatty. -->') }}</p>
```